code/datums/autolathe/arms.dm

```diff
@@ -184,102 +171,8 @@
 /datum/category_item/autolathe/arms/shotgun_clip_beanbag
 	name = "4-round 12g shell clip (beanbag)"
 	path =/obj/item/ammo_magazine/clip/c12g/beanbag
-
-/* Commented out until autolathe stuff is decided/fixed. Will probably remove these entirely. -Spades
```

Comment:
almost forgot I deleted some of my own commented code which is no longer actually useful as too much has changed